pkg/rulerequest/handle.go

```diff
@@ -17,10 +17,12 @@ import (
 func PromptRun(ctx context.Context, c client.Client, dangerous bool, image string, opts client.AppRunOptions) (*apiv1.App, error) {
 	app, err := c.AppRun(ctx, image, &opts)
 	if permErr := (*client.ErrRulesNeeded)(nil); errors.As(err, &permErr) {
-		if ok, promptErr := handleDangerous(dangerous, &permErr.Permissions); promptErr != nil {
+		if ok, promptErr := handleDangerous(dangerous, permErr.Permissions); promptErr != nil {
 			return nil, fmt.Errorf("%s: %w", promptErr.Error(), err)
 		} else if ok {
-			opts.Permissions = &permErr.Permissions
+			// Transitions from they were in the Acornfile to the appInstance.Spec.Permissions. AppRun
+			// translates the permissions. Something must be going wrong here
```

Comment:
"Something must be going wrong here"?

This comment confused me

Reply:
Left over development comment. Removing.